iOS Client/DevotionsController/MBDevotionsViewController.swift

```diff
@@ -52,8 +55,15 @@ class MBDevotionsViewController: UIViewController, StoreSubscriber, UITableViewD
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let devotion = devotions[indexPath.row]
-        let cell = tableView.dequeueReusableCell(withIdentifier: "DevotionTableViewCell") ?? UITableViewCell()
-        cell.textLabel?.text = devotion.text
+        //swiftlint:disable force_cast
+        let cell = tableView.dequeueReusableCell(withIdentifier: "DevotionTableViewCell") as! DevotionTableViewCell
+        //swiftlint:enable force_cast
+        if let date = Formatters.devotionDateFormatter.date(from: devotion.devotion.date),
+            let day = Formatters.calendar?.components(.day, from: date).day,
+            let monthInt = Formatters.calendar?.component(.month, from: date) {
```

Comment:
if we move this into the devotion class as a lazy property, we don't have to repeat these computations everywhere. Maybe they will have to be optional since there's a chance of failure, or maybe we should throw in `!` and fail fast since we theoretically control the data source